app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,78 @@
+<h1 class="govuk-heading-l">Edit <%= resource_name.to_s.humanize %></h1>
```

Comment:
what is this view for?

Reply:
Its the actual view for the registration controller.  If something goes wrong with a profile update you end up here.